strings.txt

```diff
@@ -16858,3 +16858,955 @@
+    fr = 2. Appuyer sur Localisation
+    da = 2. Tryk på Placering
+    id = 2. Ketuk Lokasi
+    ko = 2. 위치 누르기
+    sv = 2. Tryck på Plats
+    tr = 2. Konuma Dokunun
+    uk = 2. Клацніть «Місцезнаходження»
+    vi = 2. Chạm mục Địa điểm
+    hu = 2. Érintse meg a Helyzet gombot
+    de = 2. Auf Standort tippen
+    fi = 2. Tik op locatie
+    cs = 2. Klepněte na položku Poloha
+    it = 2. Tocca su Localizzazione
+    nb = 2. Trykk Lokalisering
+    zh-Hant = 2. 點擊位置
+    zh-Hans = 2. 点击位置
+    th = 2. แตะตำแหน่งที่ตั้ง
+    ja = 2. 位置情報をタップします
+    ro = 2. Apăsați pe Localizare
+    ar = 2. انقر على الموقع
+    sk = 2. Kliknite na položku Lokalita
+    es = 2. Pulsa "Ubicación"
+    pl = 2. Dotknij 'Lokalizacja'
+    nl = 2. Tik op locatie
+    pt = 2. Tocar em localização
+    he = 2. געו ב"מיקום" (Location)
+
+  [location_services_disabled_3]
+    en = 3. Select Always
+    ru = 3. Выберите «Всегда»
+    fr = 3. Sélectionner Toujours
+    da = 3. Vælg Altid
+    id = 3. Pilih Selalu
+    ko = 3. 항상 선택
+    sv = 3. Välj Alltid
+    tr = 3. Her Zamanı Seçin
+    uk = 3. Виберіть «Завжди»
+    vi = 3. Chọn Luôn luôn
+    hu = 3. Válassza ki a Mindig gombot
+    de = 3. 'Immer' auswählen
+    fi = 3. Selecteer 'Altijd'
+    cs = 3. Vyberte Vždy zapnuto
+    it = 3. Seleziona Sempre
+    nb = 3. Velg Alltid
+    zh-Hant = 3. 選擇總是
+    zh-Hans = 3. 选择总是
+    th = 3. เลือก เสมอ
+    ja = 3. 「常時」を選択します
+    ro = 3. Selectați Întotdeauna
+    ar = 3. اختر دائما
+    sk = 3. Vyberte Vždy
+    es = 3. Selecciona "Siempre"
+    pl = 3. Wybierz 'Zawsze'
+    nl = 3. Selecteer 'Altijd'
+    pt = 3. Selecionar sempre
+    he = 3. בחרו ב"תמיד" (Always)
+
+  [not_now_button]
+    comment = iOS Кнопка Диалога при отключенном пермишине на поиск геопозиции
+    en = Not now
+    ru = Не сейчас
+    fr = Pas maintenant
+    da = Ikke nu
+    id = Tidak Sekarang
+    ko = 지금 아님
+    sv = Inte nu
+    tr = Şimdi değil
+    uk = Не зараз
+    vi = Không phải lúc này
+    hu = Most nem
+    de = Nicht jetzt
+    fi = Niet nu
+    cs = Později
+    it = Non ora
+    nb = Ikke nå
+    zh-Hant = 現在不要
+    zh-Hans = 现在不要
+    th = ไม่ใช่ตอนนี้
+    ja = 後で
+    ro = Nu acum
+    ar = ليس الآن
+    sk = Nie teraz
+    es = Ahora no
+    pl = Nie teraz
+    nl = Niet nu
+    pt = Agora não
+    he = לא עכשיו
+
+  [settings_button]
+    en = Settings
+    ru = Настройки
+    fr = Paramètres
+    da = Indstillinger
+    id = Pengaturan
+    ko = 설정
+    sv = Inställningar
+    tr = Ayarlar
+    uk = Налаштування
+    vi = Thiết lập
+    hu = Beállítások
+    de = Einstellungen
+    fi = İnstellingen
+    cs = Nastavení
+    it = Impostazioni
+    nb = Innstillinger
+    zh-Hant = 設置
+    zh-Hans = 设置
+    th = การตั้งค่า
+    ja = 設定
+    ro = Setări
+    ar = الإعدادات
+    sk = Nastavenia
+    es = Ajustes
+    pl = Ustawienia
+    nl = İnstellingen
+    pt = Configurações
+    he = הגדרות
+
+  [placepage_parking_surface]
+    comment = Parking type in Place Page
```

Comment:
Название строки дублирует такой комментарий, я потру их.
> On Apr 26, 2016, at 4:21 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61083372>:
> 
> > +    nb = Innstillinger
> > +    zh-Hant = 設置
> > +    zh-Hans = 设置
> > +    th = การตั้งค่า
> > +    ja = 設定
> > +    ro = Setări
> > +    ar = الإعدادات
> > +    sk = Nastavenia
> > +    es = Ajustes
> > +    pl = Ustawienia
> > +    nl = İnstellingen
> > +    pt = Configurações
> > +    he = הגדרות
> > +
> > +  [placepage_parking_surface]
> > +    comment = Parking type in Place Page
> Этот коммент надо добавить ниже в остальные варианты типов парковок.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295..454ca48abdc1a7a643a178fac869c249f100121f#r61083372>